### PR TITLE
Hn happ add zome

### PR DESCRIPTION
**Pull request by philipbeadle**

### CODE REVIEW

File: ui_src/src/resolvers.js [hunk 1]
```diff
@@ -10,21 +10,21 @@ function dnaToUiNote (noteResult) {
 export const resolvers = {
   Query: {
     getNote: async (_, { id }) =>
-      dnaToUiNote(await createZomeCall('/react-graphql/notes/get_note')({ id })),
+      dnaToUiNote(await createZomeCall('/reactGraphql/notes/get_note')({ id })),
```
Comment (robbiecarlton) [inline ui_src/src/resolvers.js]: can we change the dna instance id to something more specific? even `notes` seems fine to me. so `notes/notes/get_note` etc.

**Review by robbiecarlton — CHANGES_REQUESTED** [review]
Looks good overall. One change requested.

File: ui_src/src/resolvers.js (hunk 1 above)
Comment (philipbeadle) [reply ui_src/src/resolvers.js]: Yep, I'd like the instance to be the dna name, in this case notes.

Comment (philipbeadle) [reply ui_src/src/resolvers.js]: Done.

**Pull request merged**